README.md

```diff
@@ -853,50 +853,53 @@ You normally use this as follow up call to `uploadFiles()`. This will start your
 
 This is the most complex function call in the Bee API and has multiple options and parts.
 
-a) parameter object
+a) parameters array
 b) subscriptions
 c) subscription events
 
 
 options:
 
 ```
-bees          => array of bee system names, e.g. "xcoobee_digital_signature_detection"
-parameters    => optional: the parameters object. For each bee by bee name.
-subscriptions => optional: the subscriptions object. Specifies the subscriptions.'
-config        => optional: the config object
+bees          => array of bee system names (e.g. "xcoobee_digital_signature_detection") and their parameters
+parameters    => optional: the parameters array. For each bee by bee name.
+subscriptions => optional: the subscriptions array. Specifies the subscriptions.
+config        => optional: the config array.
 ```
 
 
-
-
-### `a` Parameters Object
+### `a` Parameters
 
 Parameters can be bee specific or apply to the overall job.
 
 Overall job parameters to be used for the hiring are specified with the `process` prefix including destinations (recipients) to which you wish to send the output of the processing.
 
 general process parameters example:
 ```
-process.userReference="myownreference"
-process.destinations=["~xcoobeeIds",{"xcoobeeId":"~jonny","accesskey":"isfnsfhis"},"emails"]
-process.Integrations.XcooBeeInbox=[{"filename": "fileinInbox.png"}]
+$options['process']['userReference'] = 'myownreference';
+$options['process']['destinations'] = [
+    ['email' => 'email@mysite.com'],
```

Comment:
we check and decide what user pass (xcoobee id or email) in code, so no need to use structure like `[['email' => 'email@mysite.com'],...]`. We should write here `['email@mysite.com', '~jonny', ...]`